notebooks/microphysics.ipynb

```diff
@@ -99,41 +59,97 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install ../../mlmicrophysics/."
+    "!pip install ../../mlmicrophysics"
```

Comment:
Can you remove this line from the notebook? It is only valid on Casper or a local machine.

Reply:
done!